src/main/webapp/index.html

```diff
@@ -5,13 +5,15 @@
   <meta charset="UTF-8">
   <title>Graphs</title>
   <link rel="stylesheet" href="style.css">
-  <script src="script.js"></script>
+  <script src="cytoscape/cytoscape.min.js"></script>
 </head>
 
-<body id="content" onload="testFunc()">
+<body id="content">
   <h1>
-    HELLO
+    Graph Visualization Tool
   </h1>
+  <div id="graph"></div>
+  <script src="script.js"></script>
```

Comment:
Instead of embedding this script in the HTML, it's a better separation of concerns if you place all JS in the <head> section and call `generateGraph()` on body load.